demos/_unit-test/late-output-error.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Atk4\Ui\Demos;
+
+use Atk4\Ui\CallbackLater;
+
+/** @var \Atk4\Ui\App $app */
+require_once __DIR__ . '/../init-app.php';
+
+$cb = CallbackLater::addTo($app);
+$cb->setUrlTrigger('trigger_err_headers_sent');
+
+$modal = \Atk4\Ui\Modal::addTo($app, ['cb' => $cb]);
+$modal->name = 'm_header';
+
+$modal->set(function () {
+    header('x-unmaanged-header: test');
+    flush();
+});
+
+$cb2 = CallbackLater::addTo($app);
+$cb2->setUrlTrigger('trigger_err_unexpected_output');
+
+$modal2 = \Atk4\Ui\Modal::addTo($app, ['cb' => $cb2]);
+$modal2->name = 'm_header';
+
+$modal2->set(function () {
+    ob_start();
+    // do we detect only flushed content or unflushed too?
+    echo 'unmanaged output';
```

Comment:
I belive this cannot be tested with JSON format, as "header sent" will be triggered first